playbooks/source-install.yml

```diff
@@ -10,7 +10,7 @@
     - name: Set up Vagrant machine for source installs
       include: ../vagrant/vagrant-pretask.yml
   roles:
-    - pulp3-postgresql
+    - pulp3-database
     - pulp3-workers
     - pulp3-resource-manager
     - pulp3-webserver
```

Comment:
Should those be renamed to `pulp-*`, too?

Reply:
Yup that's next. Just didn't want to leave it broken for too long.